src/uct/base/uct_iface.h

```diff
@@ -255,13 +267,19 @@ typedef ucs_status_t (*uct_ep_connect_to_ep_v2_func_t)(
         const uct_ep_connect_to_ep_params_t *params);
 
 
+/* Check if remote iface address is reachable */
+typedef ucs_status_t (*uct_iface_is_reachable_v2_func_t)(
```

Comment:
need to remove this

Reply:
@yosefe, can you give more detail about the comments? Thanks.